core/src/main/java/io/parsingdata/metal/Shorthand.java

```diff
@@ -115,6 +116,11 @@
     public static ComparisonExpression ltNum(final ValueExpression p) { return new LtNum(null, p); }
     public static ComparisonExpression ltNum(final ValueExpression c, final ValueExpression p) { return new LtNum(c, p); }
 
+    public final static Reducer ADD_REDUCER = new Reducer() { @Override public ValueExpression reduce(final ValueExpression l, final ValueExpression r) { return add(l, r); } };
```

Comment:
Agreed. But since #50 could be challenging, I figure it might be good to have them in the `Shorthand`. That way we promote usage of common elements which is nice if we want to replace them later.